.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[Bug] insert summary"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Versions (please complete the following information):**
+ - Release Version installed [e.g. v1.0.3]
```

Comment:
It might be helpful to provide guidance on how they can find the version. It's located in the `package.json` file here.
https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/package.json#L3

Although, with the deployment package, we don't often update the version number, so asking them what commit hash they're on might be more helpful.

Reply:
```suggestion
 - Release Version installed or commit hash [e.g. v1.0.3, 1edf4]

<!--- To get the version look at `package.json` -->
```